src/bpftrace.cpp

```diff
@@ -272,8 +293,27 @@ std::set<std::string> BPFtrace::find_wildcard_matches(
     }
 
     if (!wildcard_match(line, tokens, start_wildcard, end_wildcard))
+    {
+      if (!line.rfind("_Z", 0) || !line.rfind("____Z", 0))
```

Comment:
If this check is extracted into a function then we'd be sure it always stays in sync with the same check down in `find_symbol_matches()`. Giving it a descriptive name would also help to make it clear it's trying to match C++ symbols (self documenting!)